domains.js

```diff
@@ -0,0 +1,72 @@
+const google = require('googleapis');
+const sheets = google.sheets('v4');
+
+function getDomains(username) {
+    const credentials = credentialConfiguration();
+    const request = {
+        spreadsheetId: credentials.TASK_SHEET_ID,
+        range: 'A:ZZZ', // extreme range to get whole spreadsheet
+        auth: credentials.API_KEY
+    };
+
+    return new Promise((resolve, reject) => {
+        sheets.spreadsheets.values.get(request, function(err, response) {
```

Comment:
This is a small thing, but years of dumb mistakes working with non-native-english-speaking developers has taught be to generally avoid abbreviations like this and just make the variable `error`. This is a pretty common abbreviation, though, so not a huge deal.